Sources/Rendering/Core/AxesActor/index.js

```diff
@@ -65,34 +69,50 @@ function vtkAxesActor(publicAPI, model) {
   publicAPI.setMapper(_mapper);
 
   publicAPI.update = () => {
+    let direction;
+    if (model.xAxisInvert) {
+      direction = [-1, 0, 0];
+    } else {
+      direction = [1, 0, 0];
+    }
     const xAxis = vtkArrowSource
-      .newInstance({ direction: [1, 0, 0], ...model.config })
+      .newInstance({ direction, ...model.config })
       .getOutputData();
     if (model.config.recenter) {
       centerDataSet(xAxis);
```

Comment:
I need the inversion parameter in the shiftDataset because just changing the direction could end up in a axis being dislocated, detached from the others. This not happens with the centerDataset because of the centering operation